.github/workflows/ci.yml

```diff
@@ -164,22 +164,15 @@ jobs:
     if: github.event_name != 'release' && github.event.action != 'published'
     strategy:
       matrix:
-        package: ${{ fromJSON(needs.get-workspace-members.outputs.members) }}
         include:
+          - command: clippy
```

Comment:
If we're not running this on a per-package basis, should we be including it in the matrix vs just using a separate job?